Storage/SQL/BrowserDB.swift

```diff
@@ -151,18 +180,42 @@ open class BrowserDB {
                     success = self.createSchema(connection, schema: schema)
```

Comment:
Same.

(Note that in all of these cases you're escaping from the closure, so you do still need `success = …; return success`.)